### PR TITLE
Configure dependabot.yml to include the documentation and automations/python directories

### LINKED ISSUE #801: Configure Dependabot for `documentation/` and `automations/python/`
## Description

Depdendabot PRs are failing PR label checks. Exhibit: https://github.com/WordPress/openverse/pull/789

This will happen when Dependabot will make PRs updating depdendencies for packages that are not configured in `.github/dependabot.yml`.

## Solution

Dependabot is configured via `.github/dependabot.yml` (https://github.com/WordPress/openverse/blob/857dc4ec26fd3553aa7daddfb598978b271849ed/.github/dependabot.yml) to apply the right set of labels on the PRs. The directories `documentation/` and `automations/python/` have not been configured in that file.

### PR DESCRIPTION
**Pull request by AshwanthramKL**
## Fixes

Fixes #801  by @dhruvkb 

## Description

Configured Dependabot.yml to include the directories documentation/ and automations/python/

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -55,6 +55,34 @@ updates:
...
+    # Look for a `Pipfile` in the `/documentation` directory
+    directory: /documentation
+    # Check for updates once a month
+    schedule:
+      interval: monthly
+    labels:
+      - "💻 aspect: code"
+      - "🧰 goal: internal improvement"
+      - "🟩 priority: low"
+      - "🐍 tech: python"
```
Comment (dhruvkb) [inline .github/dependabot.yml:69]: Please also add the stack label here.
```suggestion
      - "🐍 tech: python"
      - "🧱 stack: documentation"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -55,6 +55,34 @@ updates:
...
+    # Look for a `Pipfile` in the `/automations/python` directory
+    directory: /automations/python
+    # Check for updates once a month
+    schedule:
+      interval: monthly
+    labels:
+      - "💻 aspect: code"
+      - "🧰 goal: internal improvement"
+      - "🟩 priority: low"
+      - "🐍 tech: python"
```
Comment (dhruvkb) [inline .github/dependabot.yml:83]: Please also add the stack label here.
```suggestion
      - "🐍 tech: python"
      - "🧱 stack: mgmt"
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Thanks @AshwanthramKL, this is really good! It's missing the stack labels though, so I'll approve it once you add those.

**Comment by AshwanthramKL** [comment]
> LGTM! Thanks @AshwanthramKL. While we wait for a second approval, feel free to work on another issue.

Awesome thanks.

**Review by AetherUnbound — APPROVED** [review]
This is excellent, thanks for the contribution @AshwanthramKL!

**Pull request merged**